src/main/java/teammates/ui/controller/StudentCommentsPageData.java

```diff
@@ -7,41 +8,120 @@
 import teammates.common.datatransfer.AccountAttributes;
 import teammates.common.datatransfer.CommentAttributes;
 import teammates.common.datatransfer.CourseRoster;
+import teammates.common.datatransfer.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.FeedbackQuestionDetails;
+import teammates.common.datatransfer.FeedbackResponseAttributes;
+import teammates.common.datatransfer.FeedbackResponseCommentAttributes;
 import teammates.common.datatransfer.FeedbackSessionResultsBundle;
+import teammates.common.datatransfer.InstructorAttributes;
 import teammates.common.datatransfer.StudentAttributes;
+import teammates.common.util.TimeHelper;
+import teammates.ui.template.CommentRow;
+import teammates.ui.template.FeedbackResponseCommentRow;
+import teammates.ui.template.FeedbackSessionRow;
+import teammates.ui.template.QuestionTable;
+import teammates.ui.template.ResponseRow;
+import teammates.ui.template.StudentCommentsCommentRow;
+import teammates.ui.template.StudentCommentsFeedbackResponseCommentRow;
 
 /**
  * PageData: the data used in the StudentCommentsPage
  */
 public class StudentCommentsPageData extends PageData {
 
-    public String courseId;
-    public String courseName;
-    public List<String> coursePaginationList;
-    public List<CommentAttributes> comments;
-    public CourseRoster roster;
-    public String previousPageLink;
-    public String nextPageLink;
-    public String studentEmail;
-    public Map<String, FeedbackSessionResultsBundle> feedbackResultBundles;
+    private String courseId;
+    private String courseName;
+    private List<String> coursePaginationList;
+    private List<CommentAttributes> comments;
+    private CourseRoster roster;
+    private String previousPageLink;
+    private String nextPageLink;
+    private String studentEmail;
+    private Map<String, FeedbackSessionResultsBundle> feedbackResultBundles;
```

Comment:
It appears `roster` and `studentEmail` are not used on the jsp pages.
Do you need to store them here?

Also, it seems `feedbackResultBundles` is only used to determine a boolean value in those pages, so try to keep it within the method scope.